src/foam/u2/view/RichChoiceView.js

```diff
@@ -298,7 +298,7 @@ foam.CLASS({
       documentation: 'Replaces choose from placeholder with passed in string.',
       expression: function(of) {
         var plural = of.model_.plural.toLowerCase();
-        return this.CHOOSE_FROM + plural + '...';
+        return this.CHOOSE_FROM + plural + ' ...';
```

Comment:
I disagree with this change.

Reply:
we need to add space between word and Ellipsis.